threader3000.py

```diff
@@ -65,9 +70,7 @@ def threader():
       
     q = Queue()
      
-    startTime = time.time()
-     
```

Comment:
This portion of code will not be modified at this time.

Reply:
`startTime` is assigned but never used hence removing.

Reply:
I missed that.  I'll approve that change as well.